docs/api-reference/api.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: api
```

Comment:
Since this is being moved into a new section, what about also giving it a more descriptive ID? For example, "sql-ingestion-api"?

Reply:
Let's just call the API reference "HTTP API endpoints reference"